### PR TITLE
JSON parse methods

### PR DESCRIPTION
**Pull request by pskramtai**

### CODE REVIEW

File: Comparison Engine/Comparison Engine/bin/Debug/netcoreapp3.1/barInput.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                "drinkID": 1,
+                "drinkName": "Nealkoholinis alus",
+                "price": 2.0
+            },
+            {
+                "drinkID": 2,
+                "drinkName": "Nealkoholinis vynas",
+                "price": 2.5
+            }          
+        ]
```
Comment (pskramtai) [inline Comparison Engine/Comparison Engine/bin/Debug/netcoreapp3.1/barInput.json:17]: Not sure if drink names are necessary in availableDrinks list but I decided to keep them for now.

**Comment by Raeppa** [comment]
Looks good to me
Will probably need a way to write to file though in order to save the data to file

File: Comparison Engine/Comparison Engine/Data/Data.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            return bars;
+        }
+
+        public static List<Bar> GetDrinks()
+        {
+            StreamReader r = new StreamReader("drinkInput.json");
+            string json = r.ReadToEnd();
+            List<Bar> drinks = JsonConvert.DeserializeObject<List<Bar>>(json);
+            return drinks;
+        }
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Data/Data.cs:29]: Ar šitam metode neturėtų būti List<Drink>? Also might need to add some form of validation or catch exceptions in the future for reading from file

File: Comparison Engine/Comparison Engine/Comparison Engine.csproj [hunk 3]
```diff
@@ -7,4 +7,9 @@
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.8" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Comparison Engine.csproj:12]: Are we using both of these packages? Nes lygtais minėjai, kad nesigavo su Entities, tai if there's no features using them, they shouldn't be committed just yet. In that case would need to uninstall the package at the time of commit (maybe there's some less ape method of doing this, but I don't know of it yet)

**Review by ElektrosStulpas — CHANGES_REQUESTED** [review]
The only critical thing that would be nice if you could look at before merging is the thing about List<Drink> in Data.cs. Other than that, the commit should not have Bar.cs and Drink.cs and I'm not sure why it's committing gitignore again

**Review by ElektrosStulpas — APPROVED** [review]

**Pull request merged**